packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -215,6 +216,19 @@ export default ({ api }: { api: APIType }) => {
     }
   }
 
+  const fetchOpenseaStatus = function* () {
+    try {
+      const status = S.getOpenseaStatus(yield select())
+      yield put(A.fetchNftOffersMadeLoading())
+      const res: ReturnType<typeof api.getOpenseaStatus> = yield call(() => {})
+      yield put(A.fetchOpenseaStatusSuccess(res))
+      return status
```

Comment:
No need to return status if you use `getOpenseaStatus` inside the `<OpenSeaStatus />` component